packages/atomic/src/components/search/result-template-components/quickview-word-highlight/quickview-word-highlight.tsx

```diff
@@ -7,9 +8,10 @@ export class QuickviewWordHighlight {
   public occurrences = 0;
   public color: string;
   public invertColor: string;
+  public saturateColor: string;
```

Comment:
I think this should be named `saturatedColor`

Reply:
what I would like even more would be if we could rename these variables after their function rather than their color (e.g., `color`, `focusedColor`, etc)